pubspec.yaml

```diff
@@ -101,8 +101,8 @@ dependencies:
 
   pdf_render:
     git:
-      url: https://github.com/dab246/flutter_pdf_render.git
-      ref: tmail_supported
+      url: https://github.com/linagora/flutter_pdf_render.git
```

Comment:
add `Todo` note: your upstream PR was merged, but still conflict in version, will change it later

Reply:
The original project has merged PR to fix the version error but has not released it yet. So we can use the upsteam branch